terraform/aws/refactor.tf

```diff
@@ -0,0 +1,39 @@
+moved {
```

Comment:
Just read up about the `moved` config block - cool! Is this intended to stay in the repo or will it be removed after merge?

Reply:
It's intended to stay in the repo so it can be read through an upgrade (say a env got left behind for a long time, or use of a module hasn't had it's revision updated in a while but I'm beginning to think that it's not actually necessary here anymore as the move was to the global refactor that's been since undone